batch/batch/batch.py

```diff
@@ -353,6 +472,11 @@ def job_record_to_dict(app, record, running_status=None):
                 'error': traceback.format_exc(),
                 'container_statuses': {k: {} for k in tasks}
             }
+
+            if format_version > 1:
+                await log_store.write_status_file(batch_id, job_id, attempt_id, json.dumps(status))
+                status = None
```

Comment:
Doesn't this need to be the non-full status?

Reply:
Ah, maybe it' a question: should status for an error be None or {'exit_code': None, 'duration': None}.  I retract: this seems fine.